internal/app/go-filecoin/node/node.go

```diff
@@ -518,7 +546,7 @@ func (node *Node) StopMining(ctx context.Context) {
 		node.BlockMining.MiningDoneWg.Wait()
 	}
 
-	// TODO: stop node.StorageProtocol.StorageMiner
+	// TODO: stop node.StorageProtocol.StorageProvider
```

Comment:
Shouldn't we keep both TODOs?  AFAICT the storage miner is a separate component that will need to be stopped independently of stopping the storage protocol provider.

Reply:
Good catch. There was nothing really to stop for the storage provider. I added the code to stop the storage miner.